src/components/EditFeatureDrawer/SaveButton/index.tsx

```diff
@@ -115,21 +130,29 @@ export const SaveButton: React.FC<SaveButtonProps> = ({
 
       layer.getSource()?.refresh();
 
+      setSaveCompleted(true);
+      setTimeout(() => {
+        setSaveCompleted(false);
+      }, 3000);
```

Comment:
Why do we need the `setTimeout`?

Reply:
the success icon is removed after 3s. another option would be to keep it active until the form is modified again

Reply:
No, this is fine, thanks!